risc0/bigint2/src/ec/mod.rs

```diff
@@ -70,58 +56,77 @@ impl<const WIDTH: usize> WeierstrassCurve<WIDTH> {
     /// The curve as concatenated u32s
     ///
     /// Little-endian, prime then a then b
-    fn as_u32s(&self) -> &[[u32; WIDTH]; 3] {
+    pub(crate) fn as_u32s(&self) -> &[[u32; WIDTH]; 3] {
         &self.buffer
     }
 }
-impl WeierstrassCurve<EC_256_WIDTH_WORDS> {
-    /// The secp256k1 curve configuration.
-    pub const fn secp256k1() -> &'static WeierstrassCurve<EC_256_WIDTH_WORDS> {
-        SECP256K1_CURVE
-    }
-}
 
-#[derive(Clone, Debug, Eq, PartialEq, Copy)]
+/// An affine point on an elliptic curve.
+#[derive(Debug, Eq, PartialEq)]
 pub struct AffinePoint<const WIDTH: usize, C> {
     buffer: [[u32; WIDTH]; 2],
+    infinity: bool,
```

Comment:
I think this is a good clarification, thank you. Personally, I think a re-name to identity would be even better [insert pedantic math lecture about an ellipitic curve's point at infinity not being the same as infinity], but I'm content to go along with infinity if people prefer it.

Reply:
Yeah, sounds good, I'll do identity, given it would be the same name as the identity constant we have. I was just defaulting to infinity because of k256 naming